src/__tests__/testScreen.py

```diff
@@ -77,16 +77,16 @@ def compareToExpected(self, testName, actualLines):
         self.assertEqual(
             len(actualLines),
             len(expectedLines),
-            'Actual lines was %d but expected lines aws %d' % (
-                len(actualLines), len(expectedLines)),
+            'In file %s, actual lines was %d but expected lines was %d' % (
+                testName, len(actualLines), len(expectedLines)),
```

Comment:
I was kinda lazy. It would be cool if we can build the whole filepath. That way we can use fpp to pick it out. LOL. I'll change it to get the actual filepath.